packages/Algernon.package/ALGInputWrapper.class/instance/handleKeystroke..st

```diff
@@ -0,0 +1,3 @@
+as yet unclassified
+handleKeystroke: anEvent 
+	" no-op "
```

Comment:
We should probably whitelist methods we want to forward in `doesNotUnderstand` instead of blacklisting like this; it's something we forgot yesterday

Reply:
Should be done now in the newest commit. Please check.